gramex/apps/admin2/gramexadmin.py

```diff
@@ -43,13 +44,35 @@ def get_auth_conf(kwargs):
         # For DBAuth, hoist the user.column into as the id: for the URL
         user_column = auth_kwargs.get('user', {}).get('column', 'user')
         data_conf = gramex.handlers.DBAuth.clear_special_keys(
-            auth_kwargs.copy(), 'user', 'password', 'forgot', 'signup', 'template', 'delay')
+            auth_kwargs.copy(), 'rules', 'user', 'password', 'forgot',
+            'signup', 'template', 'delay')
         data_conf['id'] = user_column
         return authhandler, auth_conf, data_conf
     else:
         raise ValueError('Missing lookup: in url.%s (authhandler)' % authhandler)
 
 
+@coroutine
+def send_welcome_email(handler):
+    if handler.request.method == 'POST':
+        to = handler.get_arg('email', False)
+        if not to:
+            app_log.warning('No email address found for new user {handler.get_arg("user")}.')
```

Comment:
Use the authhandler's `.kwargs.email_column` that defaults to email -- instead of hand coding the field name as email